a10_nlbaas2oct/driver.py

```diff
@@ -375,7 +375,15 @@ def main():
         conf_lb_id_list = []
 
     if CONF.lb_id:
+        if not db_utils.get_loadbalancer_entry(n_session, CONF.lb_id):
```

Comment:
Instead of adding an extra DB call, we can just confirm that CONF.lb_id is in `lb_id_list`

Reply:
I have removed the validation since it already exists and an exception is raised for the lb-id option. It is not there for a project-id option.
Check [https://github.com/a10networks/a10-nlbaas2oct/blob/9e01e32066106ae4c332721d79c631c91e3ee214/a10_nlbaas2oct/db_utils.py#L61](https://github.com/a10networks/a10-nlbaas2oct/blob/9e01e32066106ae4c332721d79c631c91e3ee214/a10_nlbaas2oct/db_utils.py#L61)